juntagrico/templates/menu/admin_menu.html

```diff
@@ -29,56 +36,56 @@ <h5>
             </a>
             <div class="collapse" id="subman">
                 <ul class="nav flex-column subadmin-menu">
-                    {% if can_filter_subscriptions %}
+                    {% if perms.juntagrico.can_filter_subscriptions %}
                         <li class="nav-item">
                             <a class="nav-link" href="{% url 'filter-subs' %}">
                                 {% blocktrans %}Aktive{% endblocktrans %}
                             </a>
                         </li>
                     {% endif %}
-                    {% if operation_group %}
+                    {% if perms.juntagrico.is_operations_group %}
                         <li class="nav-item">
                             <a class="nav-link" href="{% url 'sub-mgmt-waitinglist' %}">
                                 {% trans "Warteliste" %}
                             </a>
                         </li>
                     {% endif %}
-                    {% if operation_group %}
+                    {% if perms.juntagrico.is_operations_group %}
```

Comment:
I know it was here before, but why do we check this for every entry of the menu individually?